### PR TITLE
add missing ppid, release and report to Process

### PR DESCRIPTION
**Pull request by lublak**
* https://nodejs.org/docs/latest-v12.x/api/process.html#process_process_ppid
 * https://nodejs.org/docs/latest-v12.x/api/process.html#process_process_release
 * https://nodejs.org/docs/latest-v12.x/api/process.html#process_process_report

### CODE REVIEW

File: src/js/node/Process.hx [hunk 1]
```diff
@@ -349,3 +364,11 @@ typedef MemoryUsage = {
 	heapTotal:Float,
 	heapUsed:Float
 }
+
+typedef Release = {
```
Comment (kLabz) [inline src/js/node/Process.hx:368]: Maybe fields other than `name` should be made optional?
> In custom builds from non-release versions of the source tree, only the name property may be present. The additional properties should not be relied upon to exist.

_also, whitespace issue_

Comment (kLabz) [reply src/js/node/Process.hx:368]: At the very least `lts` should be made optional or nullable.

File: src/js/node/Report.hx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	
+	/**
+		The signal that triggered the diagnostic report.
+	**/
+	var signal:String;
+	
+	/**
+		Writes a diagnostic report to a file.
+	**/
+	function writeReport(?filename:String, ?err:Error):Void;
```
Comment (kLabz) [inline src/js/node/Report.hx:53]: Return type should be `String` https://nodejs.org/docs/latest-v12.x/api/process.html#process_process_report_writereport_filename_err

**Review by kLabz — CHANGES_REQUESTED** [review]

**Comment by kLabz** [comment]
@nadako do you still want to review this PR?
@lublak are you still around / are you still willing to work on this PR? I added a few comments, shouldn't be much to do.

File: src/js/node/Process.hx (hunk 1 above)
Comment (lublak) [reply src/js/node/Process.hx:368]: done

**Comment by lublak** [comment]
@kLabz already done! 👍

**Comment by kLabz** [comment]
Thanks!

**Comment by lublak** [comment]
@kLabz these fields are not optional! 
grafik

**Comment by kLabz** [comment]
> In custom builds from non-release versions of the source tree, only the `name` property may be present. **The additional properties should not be relied upon to exist.**

They are, though. Not sure why they're not reflecting it in type hints while they're saying that just below. Also the syntax you used was not correct.

**Comment by lublak** [comment]
@kLabz oh. I guess you're right. I only paid attention to the type hints. With Nodejs is but also really often so that they do not get it correctly to reflect. Thanks!
And sry for my mistake i was on the rush.

**Pull request merged**